src/App.tsx

```diff
@@ -10,33 +13,23 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState(tabs[0].id);
+
+  const onTabSelected = (selectedTub: Tab) => {
```

Comment:
we can pass only id instead of whole Tab

Reply:
tests require an object